cypress/integration/page.spec.js

```diff
@@ -1748,7 +1748,7 @@ describe('', () => {
           todos.assertTitle(0, 'New title');
         });
 
-        it('should cancel if title was not changed', () => {
+        it.skip('should cancel if title was not changed', () => {
```

Comment:
Don't need to skip test
```suggestion
        it('should cancel if title was not changed', () => {
```